str_analysis/generate_gnomad_json.py

```diff
@@ -570,6 +636,43 @@ def add_histograms_and_compute_readviz_paths(df, gnomad_json, most_common_motif_
     return list(readviz_paths_to_rename), readviz_json
 
 
+def sort_keys(gnomad_json):
```

Comment:
this function was challenging for me to parse without an example json -- I'll add a few suggested comments below (optional, but might be nice to have for any other users in the future)

Reply:
Thanks for adding the comments! I've added an example to the docstring.